### DISCUSSION TITLE
Streaming video pixel data

### DISCUSSION THREAD

**Discussion by christian-schmaler-snkeos** [fo-dicom/fo-dicom#1783] [Q&A]
Hi!
I wanted to ask if it is possible to implement streaming of pixel data of video.
To be more precise, I have dicom instances of sop class VideoPhotographicImageStorage, transfer syntax MPEG4HighProfile/Level4.1
Can I access parts of the pixel data - so that I don't have to load the entire encoded video into memory first?

Thank you!

**Comment by jaime-olivares** [comment]
My suggestion is the following:
- Determine what is the offset of the pixel data, where the video content starts
- Put that information in a table or ancillary file
- In your web service, whenever the video is requested, open the file and seek the video position, start streaming from there
